tests/message/message_casttypes.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package tests.message;
+
+import "patch/go.proto";
+
+option go_package = "github.com/alta/protopatch/tests/message";
+
+message MessageWithCustomTypes {
+  string string_field = 1 [(go.field).casttype = "String"];
+  int32 int32_field = 2 [(go.field).casttype = "Int32"];
+  int64 int64_field = 3 [(go.field).casttype = "Int64"];
+  float float_field = 4 [(go.field).casttype = "Float"];
+  double double_field = 5 [(go.field).casttype = "Double"];
+  uint32 uint32_field = 6 [(go.field).casttype = "Uint32"];
+  uint64 uint64_field = 7 [(go.field).casttype = "Uint64"];
+}
+
+message MessageWithOneOfCustomType {
+  oneof one_of {
+    string string_field = 1 [(go.field).casttype = "String"];
+    int64 int64_field = 3 [(go.field).casttype = "Int64"];
+  }
+}
+
+message MessageWithCustomRepeatedType {
+  repeated string repeated_string_field = 1 [(go.field).casttype = "[]String"];
+}
+
+message MessageWithRepeatedCustomType {
```

Comment:
This message name and the previous test message name seem reversed. 

E.g. MessageWithRepeatedCustomType implies []T.